picard/ui/itemviews.py

```diff
@@ -860,21 +874,38 @@ def __init__(self, obj, sortable, *args):
         if obj is not None:
             obj.item = self
         self.sortable = sortable
-        self.setTextAlignment(1, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
-        self.setTextAlignment(7, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
-        self.setTextAlignment(8, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
+        self._sortkeys = {}
+        for column in [
+            MainPanel.LENGTH_COLUMN,
+            MainPanel.TRACKNUMBER_COLUMN,
+            MainPanel.DISCNUMBER_COLUMN,
+        ]:
+            self.setTextAlignment(column, QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter)
         self.setSizeHint(MainPanel.TITLE_COLUMN, ICON_SIZE)
 
+    def setText(self, column, text):
+        self._sortkeys[column] = None
+        return super().setText(column, text)
+
     def __lt__(self, other):
         if not self.sortable:
             return False
         column = self.treeWidget().sortColumn()
-        return self.sortkey(column) < other.sortkey(column)
+        return self.sortkey(column) > other.sortkey(column)
```

Comment:
Intentional change?

Reply:
It was, but I have to recheck this again and I probably should not have mixed this in here. On my Windows system the sort order was not matching the indication (assuming arrow down means ascending, arrow up descending order). So I suspected this was an oversight or regression. But now on my Linux laptop all looks fine. I'll revert this here and investigate the sort order difference separately. Maybe it is intentional and just how it works on Windows.

Reply:
I rmeoved this change from the PR